projects/clamav/Dockerfile

```diff
@@ -17,9 +17,21 @@
 FROM gcr.io/oss-fuzz-base/base-builder
 RUN apt-get update && apt-get install -y \
     flex bison \
-    automake autoconf pkg-config m4 libtool \
-    libssl-dev \
-    libcurl4-openssl-dev
+    python3 python3-pip \
```

Comment:
python3 and pip3 are already on image, can you remove those ? also cmake should be already there too.

Reply:
Sure, will test locally and then force-push the change

Reply:
This is unexpected, but I get a build failure for libxml2, lacking "Python.h":

```
2021-03-05 00:24:11 libxml2-2.9.10 DEBUG make[4]: Entering directory '/mussels/work/host-static/libxml2-2.9.10/python'
2021-03-05 00:24:11 libxml2-2.9.10 DEBUG CC       libxml.lo
2021-03-05 00:24:11 libxml2-2.9.10 DEBUG libxml.c:14:10: fatal error: 'Python.h' file not found
2021-03-05 00:24:11 libxml2-2.9.10 DEBUG #include <Python.h>
2021-03-05 00:24:11 libxml2-2.9.10 DEBUG ^~~~~~~~~~
2021-03-05 00:24:11 libxml2-2.9.10 DEBUG 1 error generated.
2021-03-05 00:24:11 libxml2-2.9.10 DEBUG Makefile:633: recipe for target 'libxml.lo' failed
```

Reply:
Ok, I removed `cmake` and subbed `python3` and `python3-pip` for `python3-dev` which passed locally.